portal_app_store/data_source.go

```diff
@@ -0,0 +1,20 @@
+// The portalappstore package contains the implementation of an in-memory store that stores
```

Comment:
We have `postgres/grove` as a package and `portala_app_store` as a package.

Can we make this `grove/portal` to be more idiomatic?

Reply:
Sure I can simplify it like such.

Reply:
On second thought, `portal` doesn't really convey the function of the package as its main concern is storing portal apps in-memory.

WDYT about just shorting it to `store`?

Reply:
It is done.